uniffi_bindgen/src/bindings/kotlin/templates/ErrorTemplate.kt

```diff
@@ -76,12 +76,36 @@ public object {{ e|ffi_converter_name }} : FfiConverterRustBuffer<{{ type_name }
 
     @Suppress("UNUSED_PARAMETER")
```

Comment:
here and below, I think the `@Suppress` annotations can now die?

Reply:
Yup.  Also, even though the param isn't used for flat enums the `@Supress` is still not needed since it's overriding an interface method.